README.md

```diff
@@ -104,6 +104,7 @@ vars:
   hubspot_engagement_note_enabled: false
   hubspot_engagement_task_enabled: false
   hubspot_owner_enabled: false
+  hubspot_property_enabled: false                         # Disables property and property_option tables
```

Comment:
Don't forget to bump the version in the README

Reply:
Thanks. Updated.